<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaeStatClass</class>
 <widget class="QDialog" name="SaeStatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../bicycle.qrc">:/image/function_background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 112, 0);</string>
   </property>
   <property name="text">
    <string>锁 桩 状 态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>50</y>
     <width>981</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>70</y>
     <width>981</width>
     <height>631</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 237, 194);
</string>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>65</number>
   </attribute>
   <column>
    <property name="text">
     <string>锁桩编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>锁舌收</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>锁舌伸</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>是否有车</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RC531</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电子锁</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间同步</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电压</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RTC时钟</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CAN总线</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_loopBrowse">
   <property name="geometry">
    <rect>
     <x>923</x>
     <y>720</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>循环查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manualRefresh">
   <property name="geometry">
    <rect>
     <x>837</x>
     <y>720</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>手动刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saeStatShow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_manualRefresh</tabstop>
  <tabstop>pushButton_loopBrowse</tabstop>
 </tabstops>
 <resources>
  <include location="../bicycle.qrc"/>
 </resources>
 <connections/>
</ui>
